src/test/java/com/cloudbees/hudson/plugins/folder/FolderTest.java

```diff
@@ -87,10 +87,10 @@
 
         String oldName = f.getName();
 
-        HtmlForm cfg = r.createWebClient().getPage(f, "configure").getFormByName("config");
-        cfg.getInputByName("_.name").setValueAttribute("newName");
+        HtmlForm cfg = r.createWebClient().getPage(f, "confirm-rename").getFormByName("config");
```

Comment:
The URL issue went unnoticed because this test constructs the link manually rather than clicking on the actual link that was generated.